api/v1alpha1/backend_types.go

```diff
@@ -187,8 +187,18 @@ type BackendTLSSettings struct {
 	// must be specified with at least one entry for a valid configuration. Only one of
 	// CACertificateRefs or WellKnownCACertificates may be specified, not both.
 	//
+	// Only used for DynamicResolver backends.
+	//
 	// +optional
 	WellKnownCACertificates *gwapiv1a3.WellKnownCACertificatesType `json:"wellKnownCACertificates,omitempty"`
+
+	// InsecureSkipVerify indicates whether the upstream's certificate verification
+	// should be skipped. Defaults to "false".
+	//
+	// +kubebuilder:default=false
+	// +optional
+	// +notImplementedHide
+	InsecureSkipVerify *bool `json:"insecureSkipVerify,omitempty"`
```

Comment:
im not a fan of this term, but its an industry wide term

Reply:
@envoyproxy/gateway-maintainers thoughts on `insecureSkipVerify` vs `skipValidation` ?